### PR TITLE
Beautified code, changed dicts

### PR DESCRIPTION
**Pull request by jackwluo**

### CODE REVIEW

**Comment by jackwluo** [comment]
@chriddyp

File: app.py [hunk 1]
```diff
@@ -279,22 +301,22 @@ def make_dash_table( df ):
...
-        "https://cdn.rawgit.com/plotly/dash-app-stylesheets/a3401de132a6d0b652ba11548736b1d1e80aa10d/dash-goldman-sachs-report-js.js" ]
+external_js = ["https://code.jquery.com/jquery-3.2.1.min.js",
+               "https://cdn.rawgit.com/plotly/dash-app-stylesheets/a3401de132a6d0b652ba11548736b1d1e80aa10d/dash-goldman-sachs-report-js.js"]
 
 for js in external_js:
-    app.scripts.append_script({ "external_url": js })
+    app.scripts.append_script({"external_url": js})
 
 
 # In[ ]:
```
Comment (chriddyp) [inline app.py]: Let's remove comments like these too

**Comment by chriddyp** [comment]
thanks @jackwluo ! looks a lot better. heads up @alishobeiri just for reference. once we've removed all the ipython comments, 💃

**Comment by chriddyp** [comment]
💃

**Pull request merged**